config/initializers/devise.rb

```diff
@@ -178,7 +178,7 @@
 
   # ==> Configuration for :validatable
   # Range for password length.
-  config.password_length = 6..128
+  config.password_length = 8..128
 
```

Comment:
The short minimum length does worry me, but even I am tired of hearing my own voice !

Reply:
yea, they've warn me down as well